java/com/google/re2j/MakeJavaCategories.java

```diff
@@ -0,0 +1,90 @@
+package com.google.re2j;
+
+import java.io.*;
```

Comment:
Is there a document somewhere with the expected coding standards, or with IntelliJ config files that comply with them?